installinstallmacos.py

```diff
@@ -531,15 +531,17 @@ def main():
         sys.exit('This command requires root (to install packages), so please '
                  'run again with sudo or as root.')
 
+    home_dir = os.path.expanduser("~")
     current_dir = os.getcwd()
-    if os.path.expanduser("~") in current_dir:
-        bad_dirs = ['Documents', 'Desktop', 'Downloads', 'Library']
-        for bad_dir in bad_dirs:
-            if bad_dir in os.path.split(current_dir):
-                print('Running this script from %s may not work as expected. '
-                      'If this does not run as expected, please run again from '
-                      'somewhere else, such as /Users/Shared.'
-                      % current_dir, file=sys.stderr)
+    bad_subdirs = ['Documents', 'Desktop', 'Downloads', 'Library']
+    for bad_subdir in bad_subdirs:
+        bad_dir = home_dir + os.path.sep + bad_subdir
+        if bad_dir in current_dir:
+            print('*********************************************************', file=sys.stderr)
+            print('*** Running this script from %s may not work as expected.' % current_dir, file=sys.stderr)
+            print('*** If this does not run as expected, please run again from', file=sys.stderr)
+            print('*** somewhere else, such as /Users/Shared.', file=sys.stderr)
+            print('*********************************************************', file=sys.stderr)
```

Comment:
Looks to me like these lines are longer than 80 characters. Please use pylint and make sure the lines are within the current line-lengths.

Reply:
@gregneagle Did you want me to wrap at 80, or pylint's default of 100?

Reply:
Pretty sure the existing script wraps at 80.

Reply:
Why not use the previous code formatting?
```python
                print('*********************************************************\n'
                         '*** Running this script from %s may not work as expected.\n'
                         '*** If this does not run as expected, please run again from \n'
                         '*** somewhere else, such as /Users/Shared.\n'
                         '*********************************************************'
                         % current_dir, file=sys.stderr)
```
or similar.

Reply:
@gregneagle Updated as suggested.

Most of the script wraps at 80, although there is one 85-char line (line 460). I left it alone.